frontend/src/ManageOrgUsers/InviteUsersForm.jsx

```diff
@@ -88,7 +88,7 @@ function InviteUsersForm({
                       <input
                         type="text"
                         className="form-control"
-                        placeholder={t('header.organization.menus.manageUsers.enterFirstName', 'Enter full name')}
+                        placeholder={t('header.organization.menus.manageUsers.enterFulltName', 'Enter full name')}
```

Comment:
There is a typo here  and if `organization.menus.manageUsers.enterFullName` is not available add it in `en.json`